bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -605,6 +621,15 @@ def get_arm_joint_positions(self, as_array=True):
             return arm_joint_positions
         return arm_joint_positions.tolist()
 
+    def get_new_goal_given_obj_img(self, depth, bbox):
```

Comment:
Do we need this? It seems that this function is not doing anything interested :)

Reply:
Removed